gluten-arrow/src/main/java/org/apache/gluten/vectorized/NativePlanEvaluator.java

```diff
@@ -28,10 +28,13 @@
 
 import java.nio.charset.StandardCharsets;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
 
 public class NativePlanEvaluator {
+  private static final AtomicInteger id = new AtomicInteger(0);
+  private final Runtime runtime =
+      Runtimes.contextInstance(String.format("NativePlanEvaluator-%d", id.getAndIncrement()));
```

Comment:
@zhztheplayer Is this the key change in the PR? Create different runtimes, then create different VeloxMemoryManagers.